README.md

```diff
@@ -25,6 +25,24 @@ Make sure to install dependencies:
 npm install
 ```
 
+#### Special Targets
+
+The repository includes a couple of common optional targets:
+
+- `perf` - runs micro benchmarks of a project e.g. `nx perf utils` or `nx affected -t perf`
+
+#### Special Folder
+
+The repository standards organize reusable code specific to a target in dedicated folders at project root level.
+This helps to organize and share target related code.
+
+The following optional folders can be present in a project root;
+
+- `perf` - micro benchmarks related code
+- `test` - testing related code
+- `docs` - docs related files
+- `tooling` - tooling related code
```

Comment:
I would move this to the end of the README. It should go under `### Development`, not `### Setup`. And this information is more specialized than the sections we already have, I think, from a "potential new contributor" perspective.

Reply:
introduced a `Contributing` section at the end of the doc